iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -144,6 +145,27 @@ + (NSString*)app_id {
     return app_id;
 }
 
++ (NSString*)sdk_version_raw {
+	return ONESIGNAL_VERSION;
+}
+
++ (NSString*)sdk_semantic_version {
+	
+	// examples:
+	// ONESIGNAL_VERSION = @"020402"
+	// ONESIGNAL_VERSION = @"020116"
+	// so that's 6 digits, where the first two are the major version
+	// the second two are the minor version and that last two, the patch.
+	// c.f. http://semver.org/
+	
+	if(ONESIGNAL_VERSION.length != 6){
```

Comment:
Could we remove the length check here? We don't plan to move off a 6 digit semantic version number and this would be better checked in a unit test.